keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -132,6 +142,24 @@ message CheckpointContainerResponse {}
 
 ### User Stories
```

Comment:
@adrianreber - can we add a user story around leveraging it to save costs of using GPUs (for things like training models)? This could go under Container Migration or just be an addendum to fault tolerance. 

Something along the lines of: 

With GPUs becoming a costly commodity, there is an opportunity to help customers save on costs by leveraging container checkpointing to prevent re-computation if there are any faults.

Reply:
Added. Thanks.